pkg/application/doguInstallationUseCase.go

```diff
@@ -0,0 +1,30 @@
+package application
+
+import (
+	"fmt"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domainservice"
+)
+
+type DoguInstallationUseCase struct {
+	doguRepo          domainservice.DoguInstallationRepository
+	doguDomainUseCase domainservice.DoguInstallationDomainUseCase
+}
+
+func (useCase *DoguInstallationUseCase) validateDoguHealth() error {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `(*DoguInstallationUseCase).validateDoguHealth` is unused